### DISCUSSION TITLE
Add theme toggle

### DISCUSSION THREAD

**Discussion by annuk123** [pmndrs/zustand#2765] [Ideas]
I want to add a theme toggle so users can switch between light and dark modes while comfortably reading documentation.
Thank you!

**Comment by dbritto-dev** [comment]
@annuk123 SPA or MPA?

**Reply by annuk123** [reply]
SPA

**Reply by dbritto-dev** [reply]
@annuk123 `create` should be enough

**Reply by annuk123** [reply]
I couldn't understand what you said, can you please explain me?
